Mathlib/Combinatorics/SimpleGraph/ConcreteColorings.lean

```diff
@@ -48,4 +48,76 @@ theorem chromaticNumber_pathGraph (n : ℕ) (h : 2 ≤ n) :
   · simpa only [pathGraph_two_eq_top, chromaticNumber_top] using
       hc.chromaticNumber_mono_of_embedding (pathGraph_two_embedding n h)
 
+/-- In a bicolored graph colors alternate on every path -/
+theorem pathGraph_G_Hom_coloring {α} (G : SimpleGraph α) (c : G.Coloring Bool) {n : ℕ} (hn : 1 ≤ n)
+    (hom : pathGraph n →g G) (hc0 : c (hom ⟨0, hn⟩) = true) (u : Fin n) :
+    c (hom u) = (u.val % 2 = 0) := by
```

Comment:
I'm not sure you should use `Bool`. The best would be to use an arbitrary type with two elements, but I can see how that would be annoying to use. At least I would suggest using `Prop` instead, as your statement would then be `c (hom u) ↔ u.val % 2 = 0`.